trickler/mobile/lib/widgets/weight_input.dart

```diff
@@ -94,8 +94,11 @@ class WeightInputState extends State<WeightInput> {
           signed: false
         ),
         controller: controller,
+        /// TODO(cleanup): Is this necessary?
```

Comment:
The input focus is an iOS targeted feature, because iOS doesn't have a dedicated close keyboard button. So without this, the keyboard would stay open all the time on iOS

Reply:
Does the iOS keyboard have submit and cancel buttons?

Reply:
The keyboard we are currently using on iOS only has 0-9 a period and backspace buttons. So there is no submit or cancel button.